system-variables.md

```diff
@@ -1341,16 +1349,18 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 
 ### tidb_mem_quota_query
 
-- Scope: SESSION
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
 - Default value: `1073741824` (1 GiB)
-- Range: `[-1, 9223372036854775807]`
+- Range: `[128, 137438953472]`
```

Comment:
This is specifically from https://github.com/pingcap/tidb/pull/34141

It will need another PR to remove the config file option `mem-quota-query` from our docs (does not need to be in scope here).